### PR TITLE
IT-5005 | Vue 3

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/IT-4121
https://bethink.atlassian.net/browse/IT-5005
https://github.com/bethinkpl/wnl-platform/pull/3739

### CODE REVIEW

File: lib/js/components/Banner/Banner.vue
Comment (kvas-damian) [inline lib/js/components/Banner/Banner.vue:45]: vue3 return slots as function, so we can't check `length`. Fixed `v-if` on template in stories

File: .storybook/main.js
Comment (rogatty) [inline .storybook/main.js:21]: This comment doesn't fit.

File: lib/js/components/Buttons/Button/Button.vue
Comment (rogatty) [inline lib/js/components/Buttons/Button/Button.vue:24]: Will we have to update all button uses in platform (`@click.native`)? Is it necessary because of https://github.com/vuejs/core/issues/813? We could always use `emits: ["click"]` but maybe it's better to go straight for the best solution.

File: lib/js/components/SelectList/SelectList.stories.ts
Comment (rogatty) [inline lib/js/components/SelectList/SelectList.stories.ts:18]: Something is wrong here:

Screenshot 2023-01-27 at 16 08 49

File: lib/js/components/Buttons/Button/Button.vue
Comment (kvas-damian) [reply lib/js/components/Buttons/Button/Button.vue:24]: after upgrade, `.native` modifier is ignored and all events are handled by `@click`. By removing this line, we don't emit duplicated event in this component, so migration in platform should be limited to `@click.native => @click`

File: lib/js/components/SelectList/SelectList.stories.ts
Comment (rogatty) [reply lib/js/components/SelectList/SelectList.stories.ts:18]: There is a lot of warnings about it in test logs, too.

Comment (rogatty) [reply lib/js/components/SelectList/SelectList.stories.ts:18]: `icon` prop passed to https://github.com/bethinkpl/design-system/blob/0f5267f6737eb1a363656c4cedaed9a2d386c75a/lib/js/components/Icons/Icon/Icon.vue#L36-L38 is of a `Proxy` type, so that's probably the reason it doesn't validate properly.
Screenshot 2023-02-10 at 17 07 07

**Comment by rogatty** [comment]
Nothing more to be done, we're waiting for platform migration before this can be merged.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**
